src/main/java/com/mybookfinder/enterprise/dto/Person.java

```diff
@@ -5,8 +5,9 @@
 import java.util.List;
 
 public @Data class Person {
-        private String name;
-        private String emailAddress;
-        private int studentNumber;
-        private List<String> reminders;
-    }
+    private int personId;
+    private String name;
+    private String emailAddress;
+    private int studentNumber;
+    private List<String> reminders;
+}
```

Comment:
Made the formatting more consistent with other classes, and added a unique identifier variable. This will make it easy to find and track person objects as new ones are created.
Source: PlantPlaces lab and module 3 video on Lombok